larray/core/convert.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import absolute_import, print_function
```

Comment:
is there a reason this module (or the functions from this module) are not somewhere in the io package?